webapp/stores/browser_store.jsx

```diff
@@ -160,10 +161,14 @@ class BrowserStoreClass {
         if (landingPageSeen) {
             this.setLandingPageSeen(landingPageSeen);
         }
+
+        if (selectedTeams) {
+            this.setItem('selected_teams', selectedTeams);
+        }
     }
 
     isLocalStorageSupported() {
-        if (this.checkedLocalStorageSupported !== '') {
+        if (this.checkedLocalStorageSupported !== undefined) { // eslint-disable-line no-undefined
```

Comment:
I think I'll just split it into two variables then. One if we've checked for compatibility and another if that check passed. That way it'll be really obvious what it's doing and it'll avoid any Javascript weirdness